Create statement on ActivityPub vs AT Protocol discussions (2025-09-05) (#45)
swicg/general: statements/2025-09-05 ActivityPub vs AT Protocol.md

Reviewer: Thanks for working on this, Emelia.  I completely agree that the protocol supremacy arguments are incredibly unhelpful, and think a strong statement to that effect from SWICG could be very helpful!  One of the things that really resonated with me here is the point that there there isn't going to be a "winning" protocol. And that's okay!  

It's a great point about the cross-fertilization between the ecosystems (Blacksky's approach to private data is another example, building on the local-only posts pioneered by Glitch and Hometown and now adopted widely in fedi). And it's certainly true (and very under-appreciated) that ActivityPub is much richer and more flexible than is reflected in current implementations. Similarly, a lot of experimentation in the ATmosphere highlights that current implementations are only scratching the surface of what's possible. So there are likely to be even more opportunities for mutual learning going forward.

But the toxicity of the current protocol discourse on the ActivityPub side really interferes with the synergies and potential learning in both directions (and for all I know there's toxic stuff on the AT Protocol side too that I haven't noticed).  So if SWICG can help lower the temperature, that would be great!

> # Statement on conversations pitting ActivityPub against AT Protocol

Author: ```suggestion
# Statement on discussions of ActivityPub vs AT Protocol
```

There does not have to be a “winning” protocol. We do not build a better open social web for everyone by fighting and arguing about protocol superiority. That is _**not**_ how we achieve a better open social web. Instead, we must work together, cross-pollinate and share ideas, and participate within each other's communities with respect and mutual understanding. Arguing between us only emboldens those that seek to derail and destroy efforts to build an open social web.

> The practice of collaboration outside of our own groups has a long history within the standards community, whether that is with competing companies working together on standards or protocols, or collaboration between different standards bodies like the W3C and IETF.

Author: ```suggestion
The practice of collaboration outside of our own groups has a long history within the standards community, whether that is with competing companies working together on standards or protocols, or collaboration between different standards bodies like the W3C and IETF.

There has already been cross-pollination of ideas between the people working on ActivityPub and AT Protocol. For example, AT Protocol adopted an [internet draft](https://www.ietf.org/archive/id/draft-parecki-oauth-client-id-metadata-document-03.html) that was originally written to support the ActivityPub ecosystem, and projects within ActivityPub have adopted some ideas on [content labeling](https://github.com/swicg/activitypub-trust-and-safety/issues/84) and [starter packs](https://fedidevs.com/starter-packs/) from the AT Protocol ecosystem.
```

Both ActivityPub and AT Protocol can and do co-exist. This co-existence is perhaps best emphasized by the outstanding work of [Bridgy Fed](https://fed.brid.gy/) project, which connects ActivityPub, AT Protocol, and other protocols together allowing for interoperability and community that crosses between protocols. If you wanted to summarise this letter on a t-shirt, it would be “People > Protocols > Platforms”.

This statement is a call for cooling the temperature of discussions and a reminded to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, that only emboldens and empowers those that do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).

Undersigned by the following community members:

> * Emelia Smith (Author) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)

Reviewer 2: ```suggestion
* Emelia Smith (Author) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
```

Both ActivityPub and AT Protocol can and do co-exist. This co-existence is perhaps best emphasized by the outstanding work of [Bridgy Fed](https://fed.brid.gy/) project, which connects ActivityPub, AT Protocol, and other protocols together allowing for interoperability and community that crosses between protocols. If you wanted to summarise this letter on a t-shirt, it would be “People > Protocols > Platforms”.

This statement is a call for cooling the temperature of discussions and a reminded to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, that only emboldens and empowers those that do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).

> Undersigned by the following community members:

Author: ```suggestion
### Undersigned by the following community members:
```

Both ActivityPub and AT Protocol can and do co-exist. This co-existence is perhaps best emphasized by the outstanding work of [Bridgy Fed](https://fed.brid.gy/) project, which connects ActivityPub, AT Protocol, and other protocols together allowing for interoperability and community that crosses between protocols. If you wanted to summarise this letter on a t-shirt, it would be “People > Protocols > Platforms”.

This statement is a call for cooling the temperature of discussions and a reminded to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, that only emboldens and empowers those that do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).

> ### Undersigned by the following community members:

Author: ```suggestion
### Co-signed by the following community members:
```

* Emelia Smith (Author) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)

> * Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
> * Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)

Author: ```suggestion
* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
```

> * Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)

Author: ```suggestion
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://ngerakines.me/)
```

This statement is a call for cooling the temperature of discussions and a reminded to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, that only emboldens and empowers those that do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).

### Co-signed by the following community members:

* Emelia Smith (Author) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)

> * Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://ngerakines.me/)

Author: ```suggestion
* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://ngerakines.me/)
* Robert W. Gehl (AOIR, author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
```

This statement is a call for cooling the temperature of discussions and a reminded to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, that only emboldens and empowers those that do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).

### Co-signed by the following community members:

* Emelia Smith (Author) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)

> * Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://ngerakines.me/)

Author: ```suggestion
* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)
```

This statement is a call for cooling the temperature of discussions and a reminded to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, that only emboldens and empowers those that do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).

### Co-signed by the following community members:

* Emelia Smith (Author) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)

> * Robert W. Gehl (author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)

Author: ```suggestion
* Robert W. Gehl (author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)

```

There has already been cross-pollination of ideas between the people working on ActivityPub and AT Protocol. For example, AT Protocol adopted an [internet draft](https://www.ietf.org/archive/id/draft-parecki-oauth-client-id-metadata-document-03.html) that was originally written to support the ActivityPub ecosystem, and projects within ActivityPub have adopted some ideas on [content labeling](https://github.com/swicg/activitypub-trust-and-safety/issues/84) and [starter packs](https://fedidevs.com/starter-packs/) from the AT Protocol ecosystem.

Both ActivityPub and AT Protocol can and do co-exist. This co-existence is perhaps best emphasized by the outstanding work of [Bridgy Fed](https://fed.brid.gy/) project, which connects ActivityPub, AT Protocol, and other protocols together allowing for interoperability and community that crosses between protocols. If you wanted to summarise this letter on a t-shirt, it would be “People > Protocols > Platforms”.

> This statement is a call for cooling the temperature of discussions and a reminded to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, that only emboldens and empowers those that do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).

Author: ```suggestion
This statement is a call for cooling the temperature of discussions and a reminder to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, that only emboldens and empowers those that do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).
```

# Statement on discussions of ActivityPub vs AT Protocol

Date: 5th September 2025

> Over the past few months in particular, the [Social Web Community Group](https://www.w3.org/community/socialcg/) has seen an uptick in discussions that have been arguing protocol superiority and creating conflict between [ActivityPub](https://www.w3.org/TR/activitypub/) and [AT Protocol](https://atproto.com/), or trying to promote one over the other. These discussions have generally not been productive, created contention within the community that stands in the way of collaboration, and been a hotbed for conflict, disagreements, and misinformation. There has often been significant biases exhibited within these conversations.

Author: ```suggestion
Over the past few months in particular, the [Social Web Community Group](https://www.w3.org/community/socialcg/) has seen an increase in heated discussions online that have been arguing protocol superiority and creating conflict between [ActivityPub](https://www.w3.org/TR/activitypub/) and [AT Protocol](https://atproto.com/), or trying to promote one over the other. These discussions have generally not been productive, created contention within the community that stands in the way of collaboration, and been a hotbed for conflict, disagreements, and misinformation. There has often been significant biases exhibited within these conversations.
```

@erlend-sh is this clearer?

### Co-signed by the following community members:

* Emelia Smith (Author) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)
* Robert W. Gehl (author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)
* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)

> 

Author: ```suggestion
* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)

```

### Co-signed by the following community members:

* Emelia Smith (Author) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)
* Robert W. Gehl (author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)
* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)
* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)

> 

Reviewer 3: * Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/) — and to note that we also have positive experiences with a plurality of approaches all learning from and strengthening each other across the technical specifications [published by the first Social Web WG](https://www.w3.org/wiki/Socialwg#Specifications).

> 

Reviewer 3: ```suggestion
* Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/) — and to note that we also have positive experiences with a plurality of approaches all learning from and strengthening each other across the technical specifications [published by the first Social Web WG](https://www.w3.org/wiki/Socialwg#Specifications).
```

Author: I'd rather keep the signature list small, but once we publish this more broadly (I'm thinking tomorrow) feel free to reference it in a post on your site with this statement?
```suggestion
* Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)

```

### Co-signed by the following community members:

* Emelia Smith (Author) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)
* Robert W. Gehl (author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)
* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)
* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)
* Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)

> 

Author: ```suggestion
* Bailey Townsend (Independent AT Protocol Developer) [@baileytownsend.dev](https://bsky.app/profile/baileytownsend.dev)

```

* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)
* Robert W. Gehl (author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)
* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)
* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)
* Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)
* Bailey Townsend (Independent AT Protocol Developer) [@baileytownsend.dev](https://bsky.app/profile/baileytownsend.dev)

> 

Author: ```suggestion
* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)

```

* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)
* Robert W. Gehl (author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)
* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)
* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)
* Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)
* Bailey Townsend (Independent AT Protocol Developer) [@baileytownsend.dev](https://bsky.app/profile/baileytownsend.dev)
* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)
* Johannes Ernst (Dazzle Labs, FediForum) [@j12t@j12t.social](https://j12t.social/@j12t) [@j12t.org](https://j12t.org)

> 

Author: ```suggestion
* James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) (@[jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))

```

* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)
* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)
* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)
* Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)
* Bailey Townsend (Independent AT Protocol Developer) [@baileytownsend.dev](https://bsky.app/profile/baileytownsend.dev)
* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)
* Johannes Ernst (Dazzle Labs, FediForum) [@j12t@j12t.social](https://j12t.social/@j12t) [@j12t.org](https://j12t.org)
* James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) (@[jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))

> 

Reviewer 4: Chris Messina (Citizen Agency, creator of ActivityStreams) [@chrismessina.me](https://chrismessina.me)

Reviewer 5: Tim Chambers, founder of Indieweb.social @tchambers@indieweb.social

Author: ```suggestion
* Chris Messina (Citizen Agency, creator of ActivityStreams) [@chrismessina.me](https://chrismessina.me/)
Tim Chambers, founder of Indieweb.social [@tchambers@indieweb.social](https://indieweb.social/@tchambers)
```

* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)
* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)
* Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)
* Bailey Townsend (Independent AT Protocol Developer) [@baileytownsend.dev](https://bsky.app/profile/baileytownsend.dev)
* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)
* Johannes Ernst (Dazzle Labs, FediForum) [@j12t@j12t.social](https://j12t.social/@j12t) [@j12t.org](https://j12t.org)
* James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) (@[jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))
* Chris Messina (Citizen Agency, creator of ActivityStreams) [@chrismessina.me](https://chrismessina.me/)

> Tim Chambers, founder of Indieweb.social [@tchambers@indieweb.social](https://indieweb.social/@tchambers)

Author: ```suggestion
* Tim Chambers, founder of Indieweb.social [@tchambers@indieweb.social](https://indieweb.social/@tchambers)
```

> 

Author: ```suggestion
* Bryan Newbold (AT Protocol, Bluesky Social) [@bnewbold.net](https://bsky.app/profile/bnewbold.net) [@bnewbold@social.coop](https://social.coop/@bnewbold)

```

# Statement on discussions of ActivityPub vs AT Protocol

> Date: 5th September 2025

Author: ```suggestion
Date: 5th September 2025, a community letter
```

* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)
* Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)
* Bailey Townsend (Independent AT Protocol Developer) [@baileytownsend.dev](https://bsky.app/profile/baileytownsend.dev)
* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)
* Johannes Ernst (Dazzle Labs, FediForum) [@j12t@j12t.social](https://j12t.social/@j12t) [@j12t.org](https://j12t.org)
* James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) (@[jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))
* Chris Messina (Citizen Agency, creator of ActivityStreams) [@chrismessina.me](https://chrismessina.me/)
* Tim Chambers, founder of Indieweb.social [@tchambers@indieweb.social](https://indieweb.social/@tchambers)
* Bryan Newbold (AT Protocol, Bluesky Social) [@bnewbold.net](https://bsky.app/profile/bnewbold.net) [@bnewbold@social.coop](https://social.coop/@bnewbold)

> 

Reviewer 6: * David Buchanan (AT Protocol, Bluesky Social) [@retr0.id](https://bsky.app/profile/retr0.id) / `@retr0id@retr0.id`

(latter is intentionally not linkified, I don't have a public web frontend on my instance)

> 

Author: ```suggestion
* David Buchanan (AT Protocol, Bluesky Social) [@retr0.id](https://bsky.app/profile/retr0.id) / @retr0id@retr0.id

```

This statement is a call for cooling the temperature of discussions and a reminder to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, which only emboldens and empowers those who do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).

### Co-signed by the following community members:

* Emelia Smith (Author, Independent Fediverse / OAuth developer) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)

> * Robert W. Gehl (author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)

Author: ```suggestion
* Robert W. Gehl (Author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
```

* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)
* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)
* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)

> * Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)

Author: ```suggestion
* Tantek Çelik (Former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)
```

* Bailey Townsend (Independent AT Protocol Developer) [@baileytownsend.dev](https://bsky.app/profile/baileytownsend.dev)
* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)
* Johannes Ernst (Dazzle Labs, FediForum) [@j12t@j12t.social](https://j12t.social/@j12t) [@j12t.org](https://j12t.org)

> * James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) (@[jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))

Author: ```suggestion
* James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) ([@jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))
```

* Chris Messina (Citizen Agency, creator of ActivityStreams) [@chrismessina.me](https://chrismessina.me/)
* Tim Chambers (Founder of Indieweb.social) [@tchambers@indieweb.social](https://indieweb.social/@tchambers)
* Bryan Newbold (AT Protocol, Bluesky Social) [@bnewbold.net](https://bsky.app/profile/bnewbold.net) [@bnewbold@social.coop](https://social.coop/@bnewbold)

> * David Buchanan (AT Protocol, Bluesky Social) [@retr0.id](https://bsky.app/profile/retr0.id) / @retr0id@retr0.id

Author: ```suggestion
* David Buchanan (AT Protocol, Bluesky Social) [@retr0.id](https://bsky.app/profile/retr0.id) / `@retr0id@retr0.id`
```

* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)
* Johannes Ernst (Dazzle Labs, FediForum) [@j12t@j12t.social](https://j12t.social/@j12t) [@j12t.org](https://j12t.org)
* James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) ([@jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))
* Chris Messina (Citizen Agency, creator of ActivityStreams) [@chrismessina.me](https://chrismessina.me/)
* Tim Chambers (Founder of Indieweb.social) [@tchambers@indieweb.social](https://indieweb.social/@tchambers)
* Bryan Newbold (AT Protocol, Bluesky Social) [@bnewbold.net](https://bsky.app/profile/bnewbold.net) [@bnewbold@social.coop](https://social.coop/@bnewbold)
* David Buchanan (AT Protocol, Bluesky Social) [@retr0.id](https://bsky.app/profile/retr0.id) / `@retr0id@retr0.id`
* Hong Minhee (Author of Fedify and Hollo) [@hongminhee@hollo.social](https://hollo.social/@hongminhee) / [@hongminhee.org](https://bsky.app/profile/hongminhee.org)

> 

Reviewer 7: * Hailey Elizabeth (AT Protocol, Bluesky Social) [@hailey.at](https://bsky.app/profile/hailey.at)

Author: ```suggestion
* Hailey Elizabeth (AT Protocol, Bluesky Social) [@hailey.at](https://bsky.app/profile/hailey.at)

```

* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)
* Johannes Ernst (Dazzle Labs, FediForum) [@j12t@j12t.social](https://j12t.social/@j12t) [@j12t.org](https://j12t.org)
* James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) ([@jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))
* Chris Messina (Citizen Agency, creator of ActivityStreams) [@chrismessina.me](https://chrismessina.me/)
* Tim Chambers (Founder of Indieweb.social) [@tchambers@indieweb.social](https://indieweb.social/@tchambers)
* Bryan Newbold (AT Protocol, Bluesky Social) [@bnewbold.net](https://bsky.app/profile/bnewbold.net) [@bnewbold@social.coop](https://social.coop/@bnewbold)
* David Buchanan (AT Protocol, Bluesky Social) [@retr0.id](https://bsky.app/profile/retr0.id) / `@retr0id@retr0.id`
* Hong Minhee (Author of Fedify and Hollo) [@hongminhee@hollo.social](https://hollo.social/@hongminhee) / [@hongminhee.org](https://bsky.app/profile/hongminhee.org)
* Hailey Elizabeth (AT Protocol, Bluesky Social) [@hailey.at](https://bsky.app/profile/hailey.at)

> 

Author: ```suggestion
* Michael Foster (The Newsmast Foundation) 
* Jaz-Michael King (Executive Director, IFTAS) [@jaz@toot.wales](https://toot.wales/@jaz) / @jaz.co.uk
* Christine Lemmer-Webber (Executive Director of Spritely, ActivityPub co-author) [@cwebber](https://social.coop/@cwebber)

```

Christine will comment later with a public statement, but has privately let me know that she wishes to sign.

> 

Author: I'll add you in PR in a bit, as mentioned privately (sorry, you missed the merge by 13 minutes or so)
